serverless/serverless-release-notes.adoc

```diff
@@ -38,9 +39,3 @@ include::modules/serverless-rn-1-22-0.adoc[leveloffset=+1]
 include::modules/serverless-rn-1-21-0.adoc[leveloffset=+1]
 include::modules/serverless-rn-1-20-0.adoc[leveloffset=+1]
 endif::[]
-
-ifdef::openshift-enterprise[]
-// unsupported versions - remove for 4.10+ after this PR is merged
-include::modules/serverless-rn-1-19-0.adoc[leveloffset=+1]
-include::modules/serverless-rn-1-18-0.adoc[leveloffset=+1]
-endif::[]
```

Comment:
you can take this comment out re unsupported versions, but we've decided to not remove older release notes versions going forward, so please re-add 1.19 and 1.18 here